presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSql.java

```diff
@@ -53,7 +54,52 @@ protected QueryRunner createQueryRunner()
             postgreSqlServer = null;
         });
         execute("CREATE EXTENSION file_fdw");
-        return PostgreSqlQueryRunner.createPostgreSqlQueryRunner(postgreSqlServer, CUSTOMER, NATION, ORDERS, REGION);
+        return createPostgreSqlQueryRunner(
+                postgreSqlServer,
+                ImmutableMap.of(),
+                ImmutableMap.<String, String>builder()
+                        .put("metadata.cache-ttl", "10m")
```

Comment:
We used to have smoke tests without caching and distributed queries with caching. Should we run this test twice?

Reply:
Good point. Maybe we should have dedicated test with caching? like "real smoke test"?

Reply:
> Maybe we should have dedicated test with caching?

I think we should have dedicated tests for caching. Then in connector tests it should not really matter if you are using caching or not (unless you want to stress metadata access operations).

Also, if we are using caching here then we prove that caching and postgresql connector work when tests are passing. With dedicated tests we could narrow down the problem when tests would fail. Having that said, I am inclined to just tests with caching as that makes tests faster.

WDYT?

Reply:
> just tests with caching as that makes tests faster.

if caching is not on by default, we generally should not make it on by default in tests.